grpc/src/client/load_balancing/child_manager.rs

```
@@ -313,7 +313,25 @@ where
             );
             self.resolve_child_controller(channel_controller, child_idx);
         }
-        Ok(())
+    }
+
+    /// Forwards the `resolver_update` and `config` to all current children.
+    pub fn resolver_update(
+        &mut self,
+        resolver_update: ResolverUpdate,
+        config: Option<&LbConfig>,
+        channel_controller: &mut dyn ChannelController,
+    ) {
+        for child_idx in 0..self.children.len() {
+            let child = &mut self.children[child_idx];
+            let mut channel_controller = WrappedController::new(channel_controller);
+            let _ = child.policy.resolver_update(
```

Do we need this variable?

---

That's probably a classic problem of bulk status errors.  Usually, creating another Error type that represents something similar to `Vec<Box<dyn Error>>` and also provides a mergabe API to coalesce them should work.

Additionally, I am curious about what is the behavior in other languages(especially since they happen out of bounds in a work queue), do they fail silently

---

Noted. I personally, would prefer some form of aggregateable status object, but I'd be content with "pick first" for error as well.